composeApp/src/main/kotlin/io/github/couchtracker/utils/Result.kt

```diff
@@ -36,6 +36,14 @@ inline fun <I, O, E> Result<I, E>.map(f: (I) -> O): Result<O, E> = flatMap {
     Result.Value(f(it))
 }
 
+/**
+ * Runs [f] when the result is a value.
+ */
+inline fun <T, E> Result<T, E>.also(f: (T) -> Unit): Result<T, E> = map {
```

Comment:
I don't like that we are shadowing stdlib's [also](https://kotlinlang.org/api/core/kotlin-stdlib/kotlin/also.html). I think this can be called `onValue`.

Also, why are we calling `map`? I think a simple `if` would be easier, and doesn't create an additional unnecessary object.